ui/js/compareperf.js

```diff
@@ -198,9 +198,13 @@ perf.controller('CompareResultsCtrl', [
                     if (rsid == 'original') {
                         $scope.originalResultSetID = resultSet.id;
                         $scope.originalTimestamp = resultSet.push_timestamp;
+                        $scope.originalauthor = resultSet.author;
```

Comment:
You should use camel case here. I.e. originalauthor -> originalAuthor

Same for the other 3 new variables.